src/scss/_variables.scss

```diff
@@ -91,12 +91,24 @@
 //
```

Comment:
It would make much more sense, but due to the way that sass variable inheritance works, it ends up being easier for us to just maintain our own custom copy of _variables.scss.  Doing so makes sure that:
a) we always have access to Bootstrap variables when adding or modifying other variables (e.g. we don't run into issues with trying to use variables that aren't yet defined), and
b) changes we make to Bootstrap-specific variables cascade into other dependent variables.

Bootstrap actually started having custom-build users define a _custom.scss file that imported after Bootstrap's _variables.scss in an earlier alpha version, but ended up reverting the change due to variable overrides no longer working properly.  https://github.com/twbs/bootstrap/pull/21603